README.md

```diff
@@ -1,4 +1,4 @@
-[![CircleCI](https://circleci.com/gh/mne-tools/mne-study-template.svg?style=svg)](https://circleci.com/gh/mne-tools/mne-study-template)
+[![CircleCI](https://circleci.com/gh/brainthemind/CogBrainDyn_MEG_Pipeline.svg?style=svg)](https://circleci.com/gh/brainthemind/CogBrainDyn_MEG_Pipeline)
```

Comment:
You should not do this. Forks should not change upstream CIs

Reply:
that pull request was a mistake. sorry.

Reply:
how do I undo it?

Reply:
Close pr. That’s it